src/commands/database/list.mjs

```diff
@@ -1,33 +1,98 @@
 //@ts-check
 
+import { FaunaError } from "fauna";
+
 import { container } from "../../cli.mjs";
-import { commonQueryOptions } from "../../lib/command-helpers.mjs";
+import { throwForError } from "../../lib/fauna.mjs";
 import { FaunaAccountClient } from "../../lib/fauna-account-client.mjs";
 
-async function listDatabases() {
-  const logger = container.resolve("logger");
+// Narrow the output fields based on the provided flags.
+const getOutputFields = (argv) => {
+  if (!argv.secret && !argv.database) {
+    // If we are listing top level databases the region group
+    // needs to be included as database names can be re-used across
+    // regions.
+    return ["name", "region_group"];
+  }
+  return ["name"];
```

Comment:
why are we choosing not to include has `has_children`, `region_group` and `path`?

I realize that the core APIs can't give us this at this time.

Is it strictly for that reason?

Reply:
Yeah I've included `region_group` if listing at the top level as that is needed to disambiguate databases with the same name across region groups. Otherwise I've just kept it to name to be consistent when listing from both APIs. Theoretically you also know the region and path if listing with the `--database` flag.